tck/old-tck/source/src/com/sun/ts/tests/jsf/spec/el/managedbean/common/TestServlet.java

```diff
@@ -29,15 +30,16 @@
 import com.sun.ts.tests.jsf.common.servlets.HttpTCKServlet;
 import com.sun.ts.tests.jsf.common.util.JSFTestUtil;
 
+
 import jakarta.faces.application.Application;
 import jakarta.faces.context.ExternalContext;
 import jakarta.faces.context.FacesContext;
+import jakarta.el.ELManager;
```

Comment:
Did you intend to make an update in this class that actually uses ELManager?  If not, this import is not needed.